napari/components/_viewer/model.py

```diff
@@ -217,30 +217,38 @@ def to_svg(self, file=None, canvas_shape=None):
             either a str or bytes object representing a path, or an object
             implementing the `os.PathLike` protocol. If passed the svg will be
             written to this file
-        canvas_shape : 2-tuple, optional
-            Shape of SVG canvas to be generated. If not specified, takes the
-            shape of the last two dimensions of the view
+        view_box : 4-tuple, optional
+            View box of SVG canvas to be generated specified as `min-x`,
+            `min-y`, `width` and `height`. If not specified, calculated
+            from the last two dimensions of the view.
 
         Returns
         ----------
         svg : string
             String with the svg specification of the currently viewed layers
         """
 
-        if canvas_shape is None:
-            canvas_shape = self._calc_max_shape()[-2:]
+        if view_box is None:
+            min_shape = np.array(self._calc_min_shape()[-2:])
+            max_shape = np.array(self._calc_max_shape()[-2:])
+            range = max_shape - min_shape
```

Comment:
since you're only converting the above to arrays to leverage broadcasted subtraction, i would just use `np.subtract(max_shape, min_shape)` instead

Reply:
fixed